configs/configgen.sh

```diff
@@ -3,14 +3,19 @@
 set -e
 
 SCRIPT_DIR=`dirname $0`
-BUILD_DIR=$2/configgen
+BUILD_DIR=/tmp/configgen
+OUT_DIR=$1
```

Comment:
To remove dependence on `/tmp`, e.g. to avoid possible conflicts when not running sandboxed, it's probably better to do something like:

```
BUILD_DIR="$1"/build
OUT_DIR="$1"/out
```